public/src/main/java/com/revenuecat/purchases/models/SubscriptionOptions.kt

```diff
@@ -0,0 +1,104 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import androidx.annotation.VisibleForTesting
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class SubscriptionOptions(
+    private val subscriptionOptions: List<SubscriptionOption>
+    ) : List<SubscriptionOption> by subscriptionOptions, Parcelable {
+
+    /**
+     * Finds the base plan [SubscriptionOption].
```

Comment:
nit: instead of "finds", maybe we should just say "the base plan", since it's not a function. same for the rest